packages/g/gd-sprest-bs.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "gd-sprest-bs",
+  "filename": "gd-sprest-bs-icons.min.js",
```

```suggestion
  "filename": "gd-sprest-bs.min.js",
```
I'm not familiar with this library, but should this file be suggested as the default instead?

---

No, this is correct. The bootstrap icons were later added to v4, but is still made optional due to the size of the library. Once bootstrap v5 is released, I'll bundle the icons and re-update this package definition.

---

👍